### PR TITLE
HPG's LLH Counter Insurgency modpack

### PR DESCRIPTION
**Pull request by kevotosmart**
Support for this mod pack

Here is the link 
[reddit thread](https://www.reddit.com/r/DCSExposed/comments/m5uje8/insurgent_pack_work_in_progress_impressions/)
[Download](http://www.hollowpointgames.com/LLH_Counter_Insurgency.rar)

### CODE REVIEW

File: game/factions/faction.py [hunk 1]
```diff
@@ -346,6 +346,14 @@ def apply_mod_settings(self, mod_settings: ModSettings) -> None:
             self.remove_preset("SA-23/S-300VM")
             self.remove_preset("SA-17")
             self.remove_preset("KS-19")
+        # llh mod pack
+        if not mod_settings.llhmodpack:
+            self.remove_preset("HPG_Ural_Isis_Troops")
```
Comment (RndName) [inline game/factions/faction.py]: `remove_preset` is the wrong function for it. What you're looking for is `remove_vehicle`. 
The remove preset will remove all vehicles coming into the faction by the use of "preset_group". This is for example used for complex sam systems which include a set of vehicles.

File: pydcs_extensions/llhmodpack/llhmodpack.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
+# Requires French Pack mod :
```
Comment (RndName) [inline pydcs_extensions/llhmodpack/llhmodpack.py]: Is there really a requirement to have French Pack installed? If it is so, this should be stated somewhere. Extra points if you disable the checkbox and only let it be checked if frenchpack is also checked if there is a dependency between those mods

File: pydcs_extensions/llhmodpack/llhmodpack.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+@vehiclemod
+class HPGToyotaISISDShK(unittype.VehicleType):
+    id = "HPGToyotaISISDShK"
+    name = "*LLH Toyota ISIS DShk"
+    detection_range = 0
+    threat_range = 0
+    air_weapon_dist = 0
+
+
+## FORTIFICATIONS
```
Comment (RndName) [inline pydcs_extensions/llhmodpack/llhmodpack.py:64]: Are these vehicles? Beside that they are added to the list of vehicles they dont have a corresponding unit.yaml for them

**Review by RndName — CHANGES_REQUESTED** [review]

File: pydcs_extensions/llhmodpack/llhmodpack.py (hunk 3 above)
Comment (kevotosmart) [reply pydcs_extensions/llhmodpack/llhmodpack.py:64]: Yes they are vehicles, and they do have corresponding unit.yamls for them. Not sure if I’m understanding

File: pydcs_extensions/llhmodpack/llhmodpack.py (hunk 2 above)
Comment (kevotosmart) [reply pydcs_extensions/llhmodpack/llhmodpack.py]: No requirements, just a typo.

File: game/factions/faction.py (hunk 1 above)
Comment (kevotosmart) [reply game/factions/faction.py]: resolved

**Pull request closed without merging**